pkg/core/config/schema.go

```diff
@@ -166,7 +167,26 @@ var (
 		"type": "object",
 		"additionalProperties": false,
 		"properties": {
-			"css": { "type": "string" }
+			"css": { "type": "string" },
+			"country_calling_code": { "$ref": "#AuthUICountryCallingCodeConfiguration" }
+		}
+	},
+	"CountryCallingCode": {
+		"$id": "#CountryCallingCode",
+		"type": "string",
+		"enum": %s
```

Comment:
I think it's better not to embed all codes in the schema, since it would bloat the schema greatly. Also, I'm not sure if the codes are exhaustive and will not be changed. Maybe use a numeric regex instead?

Reply:
Writing a regex for it is not an good idea. Maybe make it a format instead?

Reply:
I'm thinking of just a simple regex like `^\d+$`, since it seems likely that our list of calling codes is not always correct in future.